include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -287,6 +287,13 @@ class argument_parser
                                std::string const & desc,
                                validator_type validator = validator_type{}) // copy to bind rvalues
     {
+        if (positional_option_list_was_added)
```

Comment:
As the variable was set in an `if constexpr` statement the variable itself could be compile time constant and then also this if expression can become constexpr?

Reply:
No, because I cannot track the order of functions calls at compile time and if I model the `positional_option_list_was_added` as a constexpr variable, I set its state once and than I cannot change it :/